tests/Sinch.Tests/e2e/Voice/CalloutTests.cs

```diff
@@ -83,7 +83,8 @@ public async Task CustomRequest()
                     .Parse("{\"action\":{\"name\":\"connectPstn\",\"number\":\"46000000001\",\"maxDuration\":90}}")!
                     .AsObject()!,
                 Ace = JsonNode.Parse("{}")!.AsObject(),
-                Pie = "https://your-application-server-host/application"
+                Pie = JsonNode
```

Comment:
Can you also provide an example for the case where the value is a URL string ?

Reply:
Thanks
Would the previous line still be valid when a user wants to use a simple string? Or do they have to write `Pie = JsonNode.Parse("https://your-application-server-host/application")`

Reply:
yeap, JsonNode has an implicit converstion from string.